package.json

```diff
@@ -33,7 +33,9 @@
   ],
   "author": "Jason Merino",
   "license": "MIT",
-  "dependencies": {},
+  "dependencies": {
+    "lodash": "^4.14.2"
```

Comment:
The lodash authors have broken up each function of lodash into its own module on npm. Since we are only going to use `.merge()` let's import that one into the project instead of the whole library.

```sh
npm i --save lodash.merge
```

And then in the src/index.js file.

```js
import merge from 'lodash.merge';
```